formatTest/typeCheckedTests/expected_output/mlSyntax.re

```diff
@@ -100,3 +100,9 @@ let module EM = {
 };
 
 exception Ealias = EM.E;
+
+let switch_ = "switch";
+
+let pub_ = "public";
```

Comment:
There's some utilities in here to do some of the swapping on identifiers: [here](https://github.com/facebook/reason/blob/ad1494ce5d3cb09590be3d0305949e5138407db4/src/syntax_util.ml).